Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -479,21 +479,34 @@ lemma T_isBigO_smoothingFn_mul_asympBound :
   have hb_pos : 0 < b' := R.bi_min_div_two_pos
   rw [isBigO_atTop_iff_eventually_exists]
   obtain ⟨c₁, hc₁, h_sumTransform_aux⟩ := R.eventually_atTop_sumTransform_ge
-  filter_upwards [eventually_ge_atTop R.n₀,       -- n₀_ge_Rn₀
-      eventually_forall_ge_atTop.mpr eventually_one_sub_smoothingFn_pos,    -- h_smoothing_pos
+  filter_upwards [
+      -- n₀_ge_Rn₀
+      eventually_ge_atTop R.n₀,
+      -- h_smoothing_pos
+      eventually_forall_ge_atTop.mpr eventually_one_sub_smoothingFn_pos,
+      -- h_smoothing_gt_half
       eventually_forall_ge_atTop.mpr
-        <| eventually_one_sub_smoothingFn_gt_const (1/2) (by norm_num),    -- h_smoothing_gt_half
-      eventually_forall_ge_atTop.mpr R.eventually_asympBound_pos,            -- h_asympBound_pos
-      eventually_forall_ge_atTop.mpr R.eventually_asympBound_r_pos,          -- h_asympBound_r_pos
+        <| eventually_one_sub_smoothingFn_gt_const (1 / 2) (by norm_num),
+      -- h_asympBound_pos
+      eventually_forall_ge_atTop.mpr R.eventually_asympBound_pos,
+      -- h_asympBound_r_pos
+      eventually_forall_ge_atTop.mpr R.eventually_asympBound_r_pos,
       (tendsto_nat_floor_mul_atTop b' hb_pos).eventually_forall_ge_atTop
-        R.eventually_asympBound_pos,   -- h_asympBound_floor
-      eventually_gt_atTop 0,                                                -- n₀_pos
-      eventually_forall_ge_atTop.mpr R.eventually_one_sub_smoothingFn_r_pos,  -- h_smoothing_r_pos
-      eventually_forall_ge_atTop.mpr R.rpow_p_mul_one_sub_smoothingFn_le,    -- bound1
+        -- h_asympBound_floor
+        R.eventually_asympBound_pos,
+      -- n₀_pos
+      eventually_gt_atTop 0,
+      -- h_smoothing_r_pos
+      eventually_forall_ge_atTop.mpr R.eventually_one_sub_smoothingFn_r_pos,
+      -- bound1
+      eventually_forall_ge_atTop.mpr R.rpow_p_mul_one_sub_smoothingFn_le,
       (tendsto_nat_floor_mul_atTop b' hb_pos).eventually_forall_ge_atTop
-        eventually_one_sub_smoothingFn_pos,   -- h_smoothingFn_floor
-      eventually_forall_ge_atTop.mpr h_sumTransform_aux,                     -- h_sumTransform
-      eventually_forall_ge_atTop.mpr R.eventually_bi_mul_le_r]               -- h_bi_le_r
+        -- h_smoothingFn_floor
+        eventually_one_sub_smoothingFn_pos,
+      -- h_sumTransform
+      eventually_forall_ge_atTop.mpr h_sumTransform_aux,
+      -- h_bi_le_r
+      eventually_forall_ge_atTop.mpr R.eventually_bi_mul_le_r]
```

Comment:
Are you sure `-- h_smoothingFn_floor` applies to `eventually_one_sub_smoothingFn_pos` and not to `(tendsto_nat_floor_mul_atTop b' hb_pos)`? The new positioning of the comment on line 504 is making a statement about precisely which part of the long line the comment applies to, whereas the old one wasn't.

Reply:
I just cherry-picked Damiano's changes: @adomani do you still remember this?